include/evmc/loader.h

```diff
@@ -87,6 +87,15 @@ enum evmc_loader_error_code
  */
 evmc_create_fn evmc_load(const char* filename, enum evmc_loader_error_code* error_code);
 
+/**
+ * Exposes additional information about ::EVMC_LOADER_CANNOT_OPEN error.
+ *
+ * This function is not thread-safe and not supported on all operating systems.
+ *
+ * @return Error message or NULL if no additional information is available.
+ */
+const char* evmc_cannot_open_error();
```

Comment:
I'd say this should be named `evmc_last_error_msg()` and document that the returned pointer should never be freed. Also mention that if `null` returned then no granular error message exists and caller should rely on the error code only.

This also gives the flexibility to use a static buffer if Windows would return a heap pointer.

Reply:
Ok, make sense.